Reference_ET_Part_1.py

```diff
@@ -0,0 +1,69 @@
+import math
+import numpy as np
+import re
+
+elevation = 300
+t_max = 40
+t_min = 20
+t_mean = np.mean([t_max, t_min])
+hum_max = 70
+hum_min = 50
+hum_mean = 60
+cloud_cover = 45 
+
+
+P = 101.3 * ((293-0.0065*elevation)/293) ** 5.26            # Atmospheric pressure
+λ = 2.54                                                    # Latent heat of vaporization = 2.450 MJ/kg
+γ = (0.665/1000)* P                                         # Psychrometric Constant = 0.065 kPa/⁰C
+
+
+## Calculating Mean Saturation Vapour Pressure
+
+# Saturation  Vapour Pressure at Max temp
+e_tmax = 0.6108*math.exp(17.27*t_max/(t_max+237.3))
+
+# Saturation  Vapour Pressure at Min temp
+e_tmin = 0.6108*math.exp(17.27*t_min/(t_min+237.3))
+
+# Taking average of sat vapour pressure at max and min temperature
+es = np.mean([e_tmax, e_tmin])
+
+# Calculating Actual Vapour Pressure
+
+regex = '^(0|[1-9]\d*)(\.\d+)?$'
```

Comment:
Can you write a comment here explaining this regex? I don't know what that means.